sdk/appcenter-identity/src/test/java/com/microsoft/appcenter/identity/IdentityTest.java

```diff
@@ -1140,7 +1140,7 @@ public void testRemoveAccountWhenClientAppIsNull() throws Exception {
         when(mAuthTokenContext.getHomeAccountId()).thenReturn(UUIDUtils.randomUUID().toString());
         when(mAuthTokenContext.getAuthToken()).thenReturn(UUIDUtils.randomUUID().toString());
         Identity.signOut();
-        verify(publicClientApplication, never()).getAccounts(any(PublicClientApplication.AccountsLoadedListener.class));
+        verify(publicClientApplication, never()).getAccounts(any(PublicClientApplication.AccountsLoadedCallback.class));
```

Comment:
revert those changes, we don't even use this method in code, it does not make sense to verify that, there should be a separate PR to improve test logic here.

Reply:
You can simply revert your changes and accept develop changes now for this file.

Reply:
OK, I have merged it from develop branch